src/frequenz/sdk/config/_manager.py

```diff
@@ -34,6 +34,7 @@ def __init__(  # pylint: disable=too-many-arguments
         *,
         auto_start: bool = True,
         force_polling: bool = True,
+        logging_config_key: str | Sequence[str] | None = "logging",
```

Comment:
I guess we will always have this enabled.
Maybe to simplify code we could just enable this by default and not make it configurable?

Reply:
I don't think this option adds a lot of complexity, it is just a couple of extra `if`s, and gives a bit of extra flexibility to external users that might not want to allow reconfiguring the logging. And if you just want to disable that, it means you cannot use the config manager and miss on a lot of other useful features. Is not like making the `new_receiver()` method less flexible, where we have a escape hatch (just do `manager.config_channel.new_receiver()` to create your own), here you can't use the config manager at all if you want to customize only that one small part.

I don't have a strong opinion though, given the trade-offs I prefer for it to stay but I'm also open to remove it (it should be pretty easy to add it back if needed).